containers/zaas-service/Dockerfile

```diff
@@ -0,0 +1,40 @@
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project.
+#######################################################################
+
+# base image tag
+ARG ZOWE_BASE_IMAGE=latest-ubuntu
+
+FROM zowe-docker-release.jfrog.io/ompzowe/base-jdk:${ZOWE_BASE_IMAGE} AS builder
+
+##################################
+# labels
+LABEL name="ZAAS" \
+      maintainer="carson.cook@ibm.com" \
```

Comment:
Carson is probably not the maintainer anymore

Reply:
that's right, he's mentioned in all the dockerfiles, not sure who to put here instead though